### ISSUE TITLE
Profile für Kaffeespezialitäten und Konfiguration

### ISSUE THREAD

**Issue by Bongo2** [eifel-tech/ioBroker.cloudless-homeconnect#140]
Hi @eifel-tech 
Ich konnte mir jetzt eine Visualisierung und eine Sprechende Kaffeemaschine bauen :-)
Jetzt möchte ich noch über einen Button und Alexa meinen Wunschkaffee raus lassen. 
Vermutlich geht das über diesen Datenpunkt:
`Program.Beverage_Coffee.Start`
Bei meiner Kaffeemaschine Siemens EQ.9 plus kann Profile mit unterschiedlichen Kaffeespezialitäten und Konfiguration erzeugen. Das fehlt mir in den Datenpunkten.
Man müsste den Kaffee des jeweiligen Profils auswählen können oder zumindest eine Konfiguration zusammenstellen. Wie zum Beispiel: Stärke, Milchanteil, Temperatur, Menge, Intensität, Doppelzug.
In der Config sehe ich diese Werte wie zum Beispiel:
```
        "name": "ConsumerProducts.CoffeeMaker.Option.BeanAmount",
        "states": {
          "10": "VeryMild",
          "20": "Mild",
          "30": "MildPlus",
          "40": "Normal",
          "50": "NormalPlus",
          "60": "Strong",
          "70": "StrongPlus",
          "80": "VeryStrong",
          "90": "VeryStrongPlus",
          "100": "ExtraStrong",
          "110": "DoubleShot",
          "120": "DoubleShotPlus",
          "130": "DoubleShotPlusPlus",
          "140": "TripleShot",
          "150": "TripleShotPlus",
          "200": "CoffeeGround"
        },
```
Könnte man das passend zufügen?

**Comment by eifel-tech** [comment]
Hi,
ich verstehe glaub ich nicht ganz, was du meinst. 
In deinen Datenpunkten unterhalb von `Program` müsstest du doch die passenden Optionen für das jeweilige Programm finden und einstellen können. Danach dann den DP `.Start` betätigen und das Programm mit den gerade getätigten Einstellungen sollte starten. Oder meinst du etwas anderes?

**Comment by Bongo2** [comment]
Ich habe hier nur Start:
![grafik](https://github.com/user-attachments/assets/f2d26c0a-6155-44c5-83ea-d4edd578e847)

**Comment by eifel-tech** [comment]
@Bongo2 Probier mal bitte die neueste Version von hier. 
Du musst aber vorher den Adapter stoppen und alle Datenpunkte unterhalb von `383070390180003186` (das ist doch die ID deiner Kaffeemaschine oder?) löschen. Dann den Adapter aktualisieren und wieder starten. Danach sollten die Optionen in den Programmen einstellbar sein.

**Comment by Bongo2** [comment]
Nein, die ID ist eine Andere. Aber ich habe alle Datenpunkte der Kaffemaschine gelöscht, vorher Instanz gestopt. Und über die Katze installiert.
Aber es sieht noch genauso aus wie vorher. Keine Optionen!

**Comment by eifel-tech** [comment]
Ganz sicher, dass du die richtigen DPs gelöscht hast? Wenn ich das bei mir mit deiner Config mache, habe ich Optionen bei den Programmen.

**Comment by eifel-tech** [comment]
Oh Moment, du musst natürlich auch die Config neu holen. Also Instanz stoppen, DPs löschen wie oben und zusätzlich DP `info.config` leeren. Danach Instanz wieder starten. Sorry, ich war verwirrt.
